src/pages/signin/SignInPage.js

```diff
@@ -78,7 +82,9 @@ class SignInPage extends Component {
         // - AND an account did not exist or is not validated for that login
         const showResendValidationLinkForm = this.props.credentials.login && !validAccount;
 
-        const welcomeText = this.props.translate(`welcomeText.${showPasswordForm ? 'phrase4' : 'phrase1'}`);
+        const welcomeText = showResendValidationLinkForm
```

Comment:
While we are here, would you mind updating this variable name to `shouldShowResendValidationLinkForm` to match our conventions?